### PR TITLE
Add template .vscode/* settings, ignore real ones

### PR DESCRIPTION
**Pull request by andrewbranch**
**Updated**: Adds `.vscode/settings.template.json` and `.vscode/launch.template.json` and re-gitignores `.vscode/settings.json`. This way people can easily adopt the ESLint extension settings but still maintain their own additional settings.

### CODE REVIEW

**Comment by sheetalkamat** [comment]
@andrewbranch is there a way to specify multiple ts versions.. wonder if we can also add built/local as option so we can choose between LKG vs built typescript..

**Comment by andrewbranch** [comment]
@sheetalkamat it doesn’t seem like it ☹️ Before we had this file checked in, I always kept my setting on `built/local`. I’ve accidentally included that change in a couple PRs now.

File: .vscode/settings.template.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+// Rename this file 'settings.json' or merge its
+// contents into your existing settings.
```
Comment (orta) [inline .vscode/settings.template.json]: Maybe worth a note in https://github.com/microsoft/TypeScript/blob/master/CONTRIBUTING.md ?

Comment (andrewbranch) [reply .vscode/settings.template.json]: Done

**Pull request merged**